### PR TITLE
It's GOOFY, NOT EXAMPLE

### PR DESCRIPTION
**Pull request by TheBunnyMan123**
I changed ExamplePlugin to GoofyPlugin, and fixed IntelliJ bullshit along the way

### CODE REVIEW

File: fabric/src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -4,39 +4,42 @@
...
-      "org.figuramc.exampleplugin.ExamplePermissionsPlugin"
+      "com.thekillerbunny.goofyplugin.ExamplePermissionsPlugin"
     ],
     "figura_screen": [
-      "org.figuramc.exampleplugin.ExampleScreensPlugin"
+      "com.thekillerbunny.goofyplugin.ExampleScreensPlugin"
     ],
     "figura_vanilla_part": [
-      "org.figuramc.exampleplugin.ExampleVanillaPartPlugin"
+      "com.thekillerbunny.goofyplugin.ExampleVanillaPartPlugin"
```
Comment (PoolloverNathan) [inline fabric/src/main/resources/fabric.mod.json:34]: references deleted class
```suggestion
```

File: gradle.properties [hunk 2]
```diff
@@ -3,19 +3,19 @@ java_version=17
...
 
 # Meta Properties
 # Change these to customize your mod's metadata. archives_base_name and plugin_id should match unless an override is intended.
-plugin_id = examplefiguraplugin
-display_name = Figura Example Plugin
-description = This is an example description! Tell everyone what your plugin does!
-authors = Me, Myself and I
+plugin_id = goofyfiguraplugin
+display_name = Figura Goofy Plugin
+description = This is the dumbing ground for the Figura Goofballs!
```
Comment (PoolloverNathan) [inline gradle.properties:17]: is 'dumbing' even a word?

File: gradle.properties [hunk 3]
```diff
@@ -3,19 +3,19 @@ java_version=17
...
 minecraft_version=1.20.1
 mappings=1
-enabled_platforms=fabric,forge
+enabled_platforms=fabric
 
 # Change these to your plugin id, version number and org or maven group, it must be unique.
-archives_base_name=examplefiguraplugin
+archives_base_name=goofyfiguraplugin
 mod_version=1.0
 maven_group=org.figuramc
```
Comment (PoolloverNathan) [inline gradle.properties:11]: needs to be changed to new class
```suggestion
maven_group=com.thekillerbunny.goofyplugin
```

**Review by PoolloverNathan — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 2 above)
Comment (TheBunnyMan123) [inline gradle.properties:17]: will fix

**Review by PoolloverNathan — APPROVED** [review]
looks good to, uh, just you since I didn't actually review this time

**Pull request merged**